### PR TITLE
Master relabel snippest dli

### PR DESCRIPTION
**Pull request by dli-odoo**
task: https://www.odoo.com/web?#id=32223&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720

pad: https://pad.odoo.com/p/r.bba112e20e6a3c71f8afac9f339a5873

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/static/src/js/website.tour.banner.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ odoo.define("website.tour.banner", function (require) {
         position: "bottom",
     }, {
         trigger: "#snippet_structure .oe_snippet:eq(1) .oe_snippet_thumbnail",
-        content: _t("Drag the <i>Cover</i> block and drop it in your page."),
+        content: _t("Drag the <i>Banner</i> block and drop it in your page."),
```
Comment (fgi-odoo) [inline addons/website/static/src/js/website.tour.banner.js]: leave it to Cover

File: addons/website/data/website_data.xml [hunk 2]
```diff
@@ -303,10 +303,10 @@
 
         <!-- Snippets' Default Images (to be replaced by themes) -->
 
-        <record id="website.s_cover_default_image" model="ir.attachment">
+        <record id="website.s_banner_default_image" model="ir.attachment">
             <field name="public" eval="True"/>
-            <field name="name">Cover Snippet - Default Image</field>
-            <field name="datas_fname">s_cover_default_image.jpg</field>
+            <field name="name">Banner Snippet - Default Image</field>
```
Comment (fgi-odoo) [inline addons/website/data/website_data.xml]: don't do this change. Leave it to "Cover"

File: addons/website/static/src/js/website.tour.banner.js (hunk 1 above)
Comment (fgi-odoo) [inline addons/website/static/src/js/website.tour.banner.js]: Leave it to "Cover"

File: addons/website/static/src/js/website.tour.banner.js [hunk 3]
```diff
@@ -32,7 +32,7 @@ odoo.define("website.tour.banner", function (require) {
         position: "bottom",
     }, {
         trigger: "#snippet_structure .oe_snippet:eq(3) .oe_snippet_thumbnail",
-        content: _t("Drag another block in your page, below the cover."),
+        content: _t("Drag another block in your page, below the banner."),
```
Comment (fgi-odoo) [inline addons/website/static/src/js/website.tour.banner.js]: to cancel

File: addons/website/views/snippets.xml [hunk 4]
```diff
@@ -25,12 +25,12 @@
                 <div class="container">
                     <div class="row content">
                         <div class="carousel-content col-md-4 col-md-offset-1 mt128 mb96">
-                            <h2>Your Banner Title</h2>
+                            <h2>Your Slider Title</h2>
```
Comment (fgi-odoo) [inline addons/website/views/snippets.xml]: It's a slide. The slider is the set of slides.

File: addons/website/views/snippets.xml [hunk 5]
```diff
@@ -14,7 +14,7 @@
     </section>
```
Comment (qsm-odoo) [inline addons/website/views/snippets.xml]: The arrows on the block_slider.png image are misplaced... the left one is against the snippet, the right one is across.

File: addons/website/views/snippets.xml [hunk 6]
```diff
@@ -387,7 +387,7 @@
...
 </template>
 
-<template id="s_faq_collapse" name="Accordion">
```
Comment (qsm-odoo) [inline addons/website/views/snippets.xml]: @fgi-odoo Fabien changed this from "FAQ Collapse" to "Accordion" last year... so we are back to FAQ? :)

**Comment by qsm-odoo** [comment]
Merged with https://github.com/odoo/odoo/commit/39ae81141777071329623dbc3ca89172b960f295

**Pull request closed without merging**